Project_One/about.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<body id="grading-element-two">
+    <button id = "grading-element-one" onclick="addit()">Don't Click!!</button>
```

Comment:
works as expected...only i needed this to be on `index.html`. Can you copy it over there, even if you keep it hidden (`display: none`)?